react/week1/my-app/src/List.js

```diff
@@ -0,0 +1,8 @@
+function ListItems(props){
+    const toDoList = props.list.map((item, index) => 
+    <li key={index}>{item.description}, {item.dueDate.toDateString()}</li>
```

Comment:
Clean way of creating a date, nice!

For the key you could alternatively use a unique ID. You can find some good ideas [here](https://stackoverflow.com/questions/39549424/how-to-create-unique-keys-for-react-elements).